### DISCUSSION TITLE
240328

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#86] [팀회의]
## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
나의 캘린더 데이터 오류 수정
## 오늘 할 일
api구조 다시 바껴서 될때까지 데이터 연동.. ㅎ
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 어제 변경사항 공유

## 어제 한 일
- useTeam 대신 useParams 사용할 수 있도록 코드 수정
- 이슈 api 적용
- 공지사항 api 적용

## 오늘 할 일
- api 구조 수정 사항 반영
- 자유게시판 페이지네이션 (오늘은 꼭ㅎ!)

## 오늘의 일정 (코어 타임)
- 1:30 병원

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항
x
## 어제 한 일
team/teamId/members UI 완성
## 오늘 할 일
어제 userContext를 해결 못해서 그거 하고...?
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
members해결중인데 잘 되지않아서 오늘도 해봐야합니다
## 오늘 할 일
groupModal member해결
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
members해결
